src/HttpOverStream.Server.Owin/Once.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace HttpOverStream.Server.Owin
+{
+    public class Once<T>

```

Comment:
Will this class be needed by aspnetcore also? Maybe we should put it in the core lib?

Reply:
I don't think so (and anyway the whole aspnet core thing should be rewritten as a Kestrel transport)

Reply:
👍